BraveRewardsUI/Ads/AdsViewController.swift

```diff
@@ -297,6 +297,10 @@ extension AdsViewController {
     
     presentingController.addChild(adsViewController)
     presentingController.view.addSubview(adsViewController.view)
+		adsViewController.didMove(toParent: presentingController)
```

Comment:
Some sort of whitespace issue going on here (using tabs instead of spaces)

Reply:
Lol its still not fixed 😂

Reply:
Fixed!